en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,100 @@
 # PX4 Continuous Integration
 
-PX4 builds and testing are performed using GitHub actions and a Jenkins server.
\ No newline at end of file
+PX4 uses continuous integration (CI) to build and run tests against every pull request (PR) and release, and to generate build artifacts such as firmware, documentation updates, and various metadata files needed by other components, like QGroundControl.
+
+The integration system uses GitHub actions and a Jenkins server.
+The configuration files for these systems can be found in the [PX4-Autopilot](https://github.com/PX4/PX4-Autopilot) repo main branch:
+- [Jenkinsfile](https://github.com/PX4/PX4-Autopilot/blob/main/Jenkinsfile)
+- [.github/workflows/](https://github.com/PX4/PX4-Autopilot/tree/main/.github/workflows).
+
+The Jenkins build status can be viewed in [ci.px4.io:8080/job/PX4/](http://ci.px4.io:8080/job/PX4/)
+
+This document outlines how the different parts of the CI system are configured and connected.
+
+## Pull Requests
+
+Automated CI checks are run for every [Pull-Request (PR)](https://github.com/PX4/PX4-Autopilot/pulls).
+
+Most of the test are run using the Github workflows in [.github/workflows](https://github.com/PX4/PX4-Autopilot/tree/main/.github/workflows):
+
+- `compile_nuttx.yml`:
+  Builds NuttX flight controller board targets.
+  It can detect Flash overflows and code syntax errors,
+- `sitl_tests.yml`: 
+  Executes [MAVSDK SITL Tests](./integration_testing_mavsdk.md).
+  Detect system-level bugs in the code (e.g. VTOL doesn't return to home)
+- `mavros_mission_tests.yml`:
+  Executes mission plans using MAVROS.
+  Detects system level bugs in close, and also verifies compatibility between MAVROS and PX4.
+- `checks.yml`:
+  Runs various make targets (using `make` command).
+  The commands for each targets are documented in the [Makefile](https://github.com/PX4/PX4-Autopilot/blob/main/Makefile)
+  - `check_format`: Checks the coding style format using Clang Tidy ?
+  - `airframe_metadata`: Builds airframe XML metadata
+  - `parameters_metadata`: Builds parameter XML metadata
+  - `module_documentation`: Builds module documentation
+  - `tests`: Runs unit tests
+  - `tests_coverage`: Runs coverage tests
```

Comment:
is there a way to check the result of said coverage tests?

Reply:
Added a link for viewing past runs